src/video/SDL_sysvideo.h

```diff
@@ -402,7 +402,7 @@ struct SDL_VideoDevice
     struct SDL_EGL_VideoData *egl_data;
 #endif
     
-#if SDL_VIDEO_OPENGL_ES || SDL_VIDEO_OPENGL_ES2
+#if SDL_VIDEO_OPENGL_ES || SDL_VIDEO_OPENGL_ES2 || SDL_VIDEO_METALANGLE
```

Comment:
I think a better approach would be to do a similar thing to the `EGL`, `CGL`, `GLX`, etc. opengl context backend defines (you can see some examples by looking at how `SDL_VIDEO_OPENGL_EGL` is used) – I don't think code here needs to care whether metal-ANGLE is used to create the OpenGL ES context, since it's just an implementation of the OpenGL ES API.

Reply:
Got it, and it makes sense.
I'm going for a `SDL_VIDEO_OPENGL_METALANGLE` if it looks nice.